annotation/parser.scm

```diff
@@ -153,15 +153,15 @@ graph by mutating global variables."
 
       ;; This shouldn't happen
       (unknown (pk 'unknown unknown #false))))
-  (for-each expr->graph expr))
+  (expr->graph expr))
 
 (define* (atomese-parser port #:optional mode)
   (set! *nodes* '())
   (set! *edges* '())
   (set! *atoms* '())
   (set! *annotation* "")
   (set! *prev-annotation* "")
-  (atomese->graph
+  (for-each atomese->graph
```

Comment:
Shouldn't the caller of `atomese-parser` take care of mapping it over a list of atomese expressions?
It seems wrong to me to return a single graph when a list of expressions is passed as an argument.

Reply:
The graph is used for visualization and there can be only one graph being visualized at a time for a request. Hence, we need to get a single graph from the parser.

Reply:
@rekado maybe you can create an issue for this so it can be tracked there?

Reply:
@rekado The latest commit resolves this. Please check it.